proctor-codegen-test/build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id 'com.indeed.java-conventions'
+    id "application"
+}
+
+dependencies {
+    implementation project(':proctor-common')
+    implementation project(':proctor-consumer')
+    implementation project(':proctor-codegen')
+    implementation project(':proctor-builder')
+    implementation libs.commonsIo
+    implementation libs.jacksonDatabind
+    implementation 'org.apache.commons:commons-lang3:3.5'
+    testImplementation libs.jacksonCore
+    testImplementation libs.servletApi
+    testImplementation libs.annotationApi
+    testImplementation libs.jsr305
+    testImplementation 'com.indeed:util-core:1.0.49-1cfb888'
+    testImplementation 'com.indeed:util-varexport:1.0.49-1cfb888'
+
+}
+
+ext {
+    javaMainClass = "com.indeed.proctor.codegen.test.ProctorGradleAutogenTask"
+}
+
+application {
+    mainClassName = javaMainClass
+}
+
+task runWithJavaExec(type: JavaExec) {
```

Comment:
What's the purpose of this?

Reply:
Ah, is this because there isn't a main method available to run the code generator? Makes sense. That's why we had to make the annoying ant-task-runner internally so we could call the proctor codegen using its ant entrypoint.